config/initializers/jwt.rb

```diff
@@ -1,42 +0,0 @@
-# rubocop:disable all
-module JWT
```

Comment:
yikes .... we had to monkeypatch JWT? glad this is going away

Reply:
Me too